api/src/classes/company.py

```diff
@@ -149,3 +145,23 @@ def delete(self) -> None:
 
         except SQLAlchemyError:
             raise HTTPException(status_code=500, detail="Unable to delete Company.") from None
+
+    def get_inventory(self) -> dict[str, Any]:
+        """Get the Company's Inventory.
+
+        :return: Company's inventory.
+        """
+        inventory: list[Inventory] | None = self.company.inventory
+
+        res: list[InventoryPublic] = []
+
+        for item in inventory:
+            data: dict[str, Any] = {
```

Comment:
A `TypedDict` should probably be created instead of `dict[str, Any]` for better type checking and auto completion